### ISSUE TITLE
_ext.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _Z27points_in_boxes_cpu_forwardN2at6TensorES0_S0_

### ISSUE THREAD

**Issue by ningqing123** [open-mmlab/mmcv#1556]
mmcv1.3.18
mmcv-full 1.3.18
mmdet 2.19.0 
pytorch==1.7.0 
torchvision==0.8.0 
torchaudio==0.7.0 
cudatoolkit=10.2

**Comment by zhouzaida** [comment]
Hi, did you install `mmcv-full` by `pip install mmcv-full` ? The pypi package of mmcv v1.3.18 has the error you mentioned.

In fact, you can install `mmcv-full` with the following command

```bash
pip install mmcv-full https://download.openmmlab.com/mmcv/dist/cu102/torch1.7.0/index.html
```

**Comment by zhouzaida** [comment]
Hi, we have released the mmcv v1.4.0 which fixed the issue.
